rapture/transport/workers.py

```diff
@@ -109,17 +109,21 @@ def scp_func(settings, filename, results):
     address = settings['address']
     username = settings['username']
     port = settings.get('port', 22)
+    destination = settings['destination']
```

Comment:
Hmm, not sure what a good default would be for this one.  Have any suggestions?  Only reason I would want to require this is I have no idea what the default directory is for paramiko sftp.  I assume the users home dir, but that might be incorrect.